pagopa/pn-frontend
# Test/pn-11507 - Implement sender dashboard unit tests

## Short description
This PR contains sender dashboard ta code

## List of changes proposed in this pull request
- StatisticsDataFactoryManager.test.ts
- StatisticsDataFactory.test.ts to verify that the factory can instantiate every StatisticsData concrete class and parses data properly

## How to test
- Run pa webapp tests and verify everything passes without errors
- Verify on sonarcloud the new code increases the test coverage appropriately
